plenum/test/malicious_behaviors_node.py

```diff
@@ -60,6 +60,23 @@ def dont_send_prepare_and_commit_to(nodes, ignore_node_name):
         node.send = types.MethodType(dont_send_prepare_commit, node)
 
 
+def dont_send_propagate(self, msg: Any, *rids, signer=None, message_splitter=None):
+    if isinstance(msg, (Propagate)):
+        if rids:
+            rids = [rid for rid in rids if rid != self.nodestack.getRemote(self.ignore_node_name).uid]
+        else:
+            rids = [self.nodestack.getRemote(name).uid for name
```

Comment:
This mechanism used, cause Delayers can't delay sending to some particular nodes